pkg/config/config.go

```diff
@@ -52,6 +51,13 @@ func (c Controller) Validate() error {
 		errs = append(errs, "newRelic.licenseKey cannot be blank")
 	}
 
+	switch c.Buildkit.Compression {
+	case CompressionGzip, CompressionStargz, CompressionZstd:
+		CompressionMethod = c.Buildkit.Compression
+	default:
+		CompressionMethod = CompressionGzip
```

Comment:
do we need to ensure that gzip is forced as the default if not otherwise set? i.e.

```
	switch c.Buildkit.Compression {
	case CompressionGzip, CompressionStargz, CompressionZstd:
		CompressionMethod = c.Buildkit.Compression
	default:
		// force gzip as the default if not otherwise set
		c.Buildkit.Compression = CompressionGzip
		CompressionMethod = CompressionGzip
	}
```